semgrep_output_v1.atd

```diff
@@ -886,57 +912,68 @@ type ci_scan_complete_stats = {
     <json repr="object">
     <python repr="dict">
     <ts repr="map">;
+
   (* This is EngineType from python, which is different from engine_kind used in this file *)
   ?engine_requested: string option;
 }
 
-type ci_scan_dependencies = (string * found_dependency list) list
-    <json repr="object">
-    <python repr="dict">
-    <ts repr="map">
-
-(* Seny by the CLI to /complete *)
-type ci_scan_complete_response = {
-  exit_code: int;
-  stats: ci_scan_complete_stats;
-  ?dependencies: ci_scan_dependencies option;
-  ?dependency_parser_errors: dependency_parser_error list option;
-  ?task_id: string option;
+type parsing_stats = {
+  targets_parsed: int;
+  num_targets: int;
+  bytes_parsed: int;
+  num_bytes: int;
 }
 
-type finding_hashes = {
-  start_line_hash: string;
-  end_line_hash: string;
-  (* hash of the syntactic_context/code contents from start_line through end_line *)
-  code_hash: string;
-  (* hash of the rule pattern with metavariables substituted in *)
-  pattern_hash: string;
-}
+(* ----------------------------- *)
+(* Findings *)
+(* ----------------------------- *)
 
-(* TODO: rewrite rule_matches.to_app_finding_format() *)
+(* TODO: rewrite rule_matches.to_app_finding_format()
+ * TODO: what's the endpoint? Where this is sent?
```

Comment:
This is sent by the CLI to the `/results` endpoint before calling `/complete`. We can have more rigorous contract testing on the app-side to check for backwards compatibility here after a version upgrade.